indico/modules/designer/templates/template.html

```diff
@@ -100,13 +100,15 @@
                             <option value="business_card" data-width="8.5" data-height="5.4">
                                 {%- trans %}Business card{% endtrans -%}
                             </option>
+                            {{ template_hook('extra-designer-tpl-page-size', landscape=true) }}
```

Comment:
You might want to pass the template itself as well, so you can know in your signal if it's for badges or posters...